lib/utils/jsonPathUtils.ts

```diff
@@ -58,4 +59,48 @@ export class JsonPathUtils {
     }
     return result;
   }
+
+  public static changePropertyNameRecursively(
+    obj: IInternalPresentationDefinition,
+    currentPropertyName: string,
+    newPropertyName: string
+  ): any {
+    console.log(newPropertyName);
+    const existingPaths: { value: unknown; path: (string | number)[] }[] = JsonPathUtils.extractInputField(obj, [
+      '$..' + currentPropertyName,
+    ]);
+    const newPd = { ...obj };
+    for (const existingPath of existingPaths) {
+      this.copyResultPathToDestinationDefinition(existingPath.path, newPd, newPd, newPropertyName);
+    }
+
+    return newPd;
```

Comment:
Why do we have an any return type, whilst we know it can only be an IInternalPresentationDefinition?

Reply:
removed